client/weapdraw.lua

```diff
@@ -102,7 +102,7 @@ end)
 Citizen.CreateThread(function()
 	while true do
 		local ped = PlayerPedId()
-		if DoesEntityExist(ped) and not IsEntityDead(ped) then
+		if DoesEntityExist(ped) and not IsEntityDead(ped) and GetPedParachuteState(ped) ~= 2 then
```

Comment:
Don't to this in same check please. Please use `if GetPedParachuteState(ped) ~= 2 then` with `else Citizen.Wait(5000) end`. It will look way better optimize.

Reply:
Wait 5 seconds if the parachute is open? How often is your parachute even open that 5 seconds of lagging your loop is going to make any sort of difference? Will not be changing the code. Kak can update if he wishes.